fluid/faster_rcnn/config.py

```diff
@@ -150,6 +150,12 @@
 # sampling ratio for roi align
 _C.sampling_ratio = 0
 
+# pooled width and pooled height 
+_C.roi_resolution = 14
+
+# spatial scale 
+_C.spatial_scale = 0.0625
```

Comment:
0.0625 is meanless, 1/16 would be better~

Reply:
Done